app/upgrades/v7/constants.go

```diff
@@ -499,3 +500,157 @@ N6juZKqdOw03MA8GA1UdEwEB/wQFMAMBAf8wCgYIKoZIzj0EAwMDaAAwZQIwc7wP
+bnRlcm5ldCBXaWRnaXRzIFB0eSBMdGQxKTAnBgNVBAMMIHNvbW1lbGllci5nb2xk
+ZW5yYXRpb3N0YWtpbmcubmV0MSUwIwYJKoZIhvcNAQkBFhZpbmZvQGdvbGRlbnN0
+YWtpbmcuY29tMB4XDTIzMDYxNTE3NDg0NloXDTI1MDYxNDE3NDg0NlowgZcxCzAJ
+BgNVBAYTAlVTMRMwEQYDVQQIDApTb21lLVN0YXRlMSEwHwYDVQQKDBhJbnRlcm5l
+dCBXaWRnaXRzIFB0eSBMdGQxKTAnBgNVBAMMIHNvbW1lbGllci5nb2xkZW5yYXRp
+b3N0YWtpbmcubmV0MSUwIwYJKoZIhvcNAQkBFhZpbmZvQGdvbGRlbnN0YWtpbmcu
+Y29tMHYwEAYHKoZIzj0CAQYFK4EEACIDYgAEoOPN2wIe3BxLKHkfPgSfqndnpSIC
+Cjb6eapS9zWzWE3BSm8PkqFX5ilIdQFre7vC5kdkx+wcP4K1ENKrG8mJiNwsfPME
+tYr0dt9yTA0u5XWcH4nhim4fuq4dixrmWFCco1MwUTAdBgNVHQ4EFgQUEZOE8j2w
+J7JFu9/Fn7SBIjf1N7AwHwYDVR0jBBgwFoAUEZOE8j2wJ7JFu9/Fn7SBIjf1N7Aw
+DwYDVR0TAQH/BAUwAwEB/zAKBggqhkjOPQQDAwNoADBlAjBDlZ/sXQ+yiu3uzCpN
+Y8sgQ+yFCuutwnC0LkmVwYybw9gtdQE79JOpwgb4vZUgX7QCMQCvA9wfEoS3LDhW
+E+m4CKm4jME2S9VYpmHN656R5mX8CQQ7Wy1G+IUcOX/QBCHLl1g=
+-----END CERTIFICATE-----
+`
+
+const CryptoCrewSubscriberCA = `-----BEGIN CERTIFICATE-----
+MIIDCDCCAo6gAwIBAgIUNUTiLAvawlgyAdvdGlYsTGmLw8kwCgYIKoZIzj0EAwMw
+gboxCzAJBgNVBAYTAkFUMRMwEQYDVQQIDApTb21lLVN0YXRlMSgwJgYDVQQKDB9T
+T0xWQSBCbG9ja2NoYWluIFNvbHV0aW9ucyBHbWJIMR4wHAYDVQQLDBVDcnlwdG9D
+cmV3IFZhbGlkYXRvcnMxJjAkBgNVBAMMHXN0ZXdhcmQtc29tbS5jY3ZhbGlkYXRv
+cnMuY29tMSQwIgYJKoZIhvcNAQkBFhVpbmZvQGNjdmFsaWRhdG9ycy5jb20wHhcN
+MjMwNDEzMDQ0NTI2WhcNMjUwNDEyMDQ0NTI2WjCBujELMAkGA1UEBhMCQVQxEzAR
+BgNVBAgMClNvbWUtU3RhdGUxKDAmBgNVBAoMH1NPTFZBIEJsb2NrY2hhaW4gU29s
+dXRpb25zIEdtYkgxHjAcBgNVBAsMFUNyeXB0b0NyZXcgVmFsaWRhdG9yczEmMCQG
+A1UEAwwdc3Rld2FyZC1zb21tLmNjdmFsaWRhdG9ycy5jb20xJDAiBgkqhkiG9w0B
+CQEWFWluZm9AY2N2YWxpZGF0b3JzLmNvbTB2MBAGByqGSM49AgEGBSuBBAAiA2IA
+BNUprvPRhwB864jaknavI4RPzgXv9BfWOdl8oIVwggpNH0w8KbsKq9nIZ9J4Viqk
+UsY+kS8hJzR2jjEK9763T40BYhEruZ+4RkiuCb/hxICI1tcB8T4Y0CFYe37YDab6
+XaNTMFEwHQYDVR0OBBYEFOuGYuXRFowcT1ZHB43fGDxeHeJMMB8GA1UdIwQYMBaA
+FOuGYuXRFowcT1ZHB43fGDxeHeJMMA8GA1UdEwEB/wQFMAMBAf8wCgYIKoZIzj0E
+AwMDaAAwZQIweTKXQJ463bUH3DhJuLfpCGrewq5GwSQMcc0DXS01OTh4UbXjod+J
+G/ksdyOwZqMGAjEA4nwkWotwVMu988zgOl/w5n+NmdH9b/qybqujnqY8VFvEk5lH
+utjI6FZqrPdhiQDw
+-----END CERTIFICATE-----
+`
+
+const DoraFactorySubscriberCA = `-----BEGIN CERTIFICATE-----
+MIICNzCCAbygAwIBAgIUYKbPgbWEtiNqgB4bao3JR5Xzgt4wCgYIKoZIzj0EAwMw
+UjEVMBMGA1UECgwMRG9yYSBGYWN0b3J5MRUwEwYDVQQLDAxEb3JhIEZhY3Rvcnkx
+IjAgBgNVBAMMGXNvbW1lbGllci5kb3JhZmFjdG9yeS5vcmcwHhcNMjMxMDMxMDkw
+MjUzWhcNMjUxMDMwMDkwMjUzWjBSMRUwEwYDVQQKDAxEb3JhIEZhY3RvcnkxFTAT
+BgNVBAsMDERvcmEgRmFjdG9yeTEiMCAGA1UEAwwZc29tbWVsaWVyLmRvcmFmYWN0
+b3J5Lm9yZzB2MBAGByqGSM49AgEGBSuBBAAiA2IABFe7uMCjv2jFQD+WmRbZZ2IF
+20JW935FImH2t1ZLny5KMTyBpcqnUdYKR8ZkJBTPZk6Uv58mKr1nXVY6f9PTmaZx
+OulptPgcZqjN3vbYWqeQ7RPqHA6uRxv+e4MLYdD7mqNTMFEwHQYDVR0OBBYEFMWX
+BT8YThoH4Ay50Psdeqx5no4uMB8GA1UdIwQYMBaAFMWXBT8YThoH4Ay50Psdeqx5
+no4uMA8GA1UdEwEB/wQFMAMBAf8wCgYIKoZIzj0EAwMDaQAwZgIxAK/znJs5LblZ
+Pz7OaQM+T3ODVMmKo/5twhd4XUZsENQgIoev6vogu1LBY/hHnQoQ7gIxAJ1qqxMt
+RscyQImrpQQiNAI8oLxTQOeKYmjZDzKAYt6aRtuDtuq07mJISbLo64J8Ug==
+-----END CERTIFICATE-----
+`
+
+const FrenchChocolatineSubscriberCA = `-----BEGIN CERTIFICATE-----
```

Comment:
Probably fine for this PR but I know at least this validator has dropped from the active set. Would it be worth culling inactive validators from this list? I can go through the list if you need me to.

Reply:
I double-checked all the validators in the upgrade against steward-registry and left in the inactive ones, just to have this mirror the current state we're working with. If you want to spend the time to verify which ones we should remove, go for it, although there isn't really a method for scrubbing dead subscribers unless they delete their subscriber themselves, I don't think. So there will likely be cruft no matter what.

Reply:
Ah, probably needed then